aiosmtpd/controller.py

```diff
@@ -346,4 +435,34 @@ def _trigger_server(self):
             s.connect(self.unix_socket)
             if self.ssl_context:
                 s = stk.enter_context(self.ssl_context.wrap_socket(s))
-            _ = s.recv(1024)
+            s.recv(1024)
+
+
+@public
+class Controller(InetMixin, BaseThreadedController):
+    def _trigger_server(self):
+        # Prevent confusion on which _trigger_server() to invoke.
+        # Or so LGTM.com claimed
+        InetMixin._trigger_server(self)
```

Comment:
LGTM.com complained if I fully rely on [Python's MRO](https://www.google.com/search?client=firefox-b-d&q=python+MRO) and don't explicitly call the superclass's `_trigger_server` method. Oh well. This also applies to lines 453-456 below.